requirements.txt

```diff
@@ -280,7 +280,7 @@ requests-oauthlib==1.3.0
     # via social-auth-core
 s3transfer==0.3.7
     # via boto3
-sentry-sdk==1.1.0
+sentry-sdk==1.4.2
```

Comment:
I believe, `.txt` is auto generated file by pip-compile, you need to touch the `.in` one

Reply:
@asadiqbal08 , Since `sentry-sdk` is not pinned in the `.in` (https://github.com/mitodl/mitxonline/blob/main/requirements.in#L32) and I kept it that way and just upgraded the `sentry-sdk` to the latest with `pip-compile` that's why there is no change in `.in`. 
Let me know if this answers your concern?

Reply:
why is it not pinned in `.in` file ?

Reply:
@asadiqbal08 After the discussion we had on a call, I've pinned sentry's version. You can take a look at the PR now as it's ready for review again.